src/views/PulseMessages/index.jsx

```diff
@@ -0,0 +1,243 @@
+        <Fragment>
+          <div className={classes.inputWrapper}>
+            <List className={classes.inputList}>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Pulse Exchange"
+                      name="pulseExchange"
+                      placeholder="exchange/<username>/some-exchange-name"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={pulseExchange}
+                    />
+                  }
+                />
+              </ListItem>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Routing Key Pattern"
+                      placeholder="#.some-interesting-key.#"
+                      name="routingKeyPattern"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={routingKeyPattern}
+                    />
+                  }
+                />
+              </ListItem>
+            </List>
+            <Tooltip title="Add Binding">
+              <IconButton
+                className={classNames(classes.iconButton, classes.plusIcon)}
+                onClick={this.handleAddBinding}>
+                <PlusIcon />
+              </IconButton>
+            </Tooltip>
+          </div>
+          <List>
+            {bindings.map(binding => (
+              <ListItem
+                className={classes.bindingListItem}
+                key={`${binding.exchange}-${binding.routingKeyPattern}`}>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography variant="body2">
+                      <code>{binding.exchange}</code> with{' '}
+                      <code>{binding.routingKeyPattern}</code>
+                    </Typography>
+                  }
+                />
+                <Tooltip title="Delete Binding">
+                  <IconButton
+                    className={classNames(
+                      classes.iconButton,
+                      classes.deleteIcon
+                    )}
+                    name={binding}
+                    onClick={() => this.handleDeleteBinding(binding)}>
+                    <DeleteIcon />
+                  </IconButton>
+                </Tooltip>
+              </ListItem>
+            ))}
+            <Toolbar>
+              <Typography variant="body2" id="tableTitle">
+                Messages
+              </Typography>
+            </Toolbar>
+            <DataTable
+              items={messages}
+              noItemsMessage="No messages received."
+            />
+          </List>
+          {listening ? (
+            <Tooltip title="Stop Listening">
+              <div className={classes.fab}>
```

Comment:
This button should probably change color when listening starts, like red.

Reply:
Yep! It does exactly that via the stopIcon class:

```jsx
stopIcon: {
  ...theme.mixins.errorIcon,
},
```

Reply:
Ah there it is, sweet.